winforms/src/toga_winforms/widgets/webview.py

```diff
@@ -75,14 +75,16 @@ def winforms_initialization_completed(self, sender, args):
             settings = self.native.CoreWebView2.Settings
             self.default_user_agent = settings.UserAgent
 
-            debug = True
+            debug = False
```

Comment:
I disabled debug mode. Was it intentionally left enabled? Does this deserve a changenote?

Reply:
Soo, this doesn't appear to be limited to WinForms. I see that a "debug mode" is enabled for macOS, iOS, and GTK as well. (I think Android drives access to DevTools with the nature of the build; debug builds enable it while release builds disable it).

I think I'll descope this for a separate effort to make this configurable.